doc/release-notes/8611-DataCommons-related-notes.md

```diff
@@ -0,0 +1,78 @@
+
+- Initial support for Globus file transfer to upload/download to/from a Dataverse managed S3 store. The current implementation disables file restriction and embargo on Globus-enabled stores.
+- Initial support for Remote File Storage. This capability, enabled via a new RemoteOverlay store type, allows a file stored in a remote system to be added to a dataset (currently only via API) with download requests redirected to the remote system. Use cases include referencing public files hosted on external web servers as well as support for controlled access managed by Dataverse (e.g. via restricted and embargoed status) and/or by the remote store.
+- Workflow (add Aday's notes here or reword to separate the Objective 2 work)
+- Support for Archiving to any S3 store using Dataverse's RDA-conformant BagIT file format (a BagPack).
+- Improved error handling and performance in archival bag creation and new options such as only supporting archiving of one dataset version.
+- Additions/corrections to the OAI-ORE metadata format (which is included in archival bags) such as referenciong the name/mimeType/size/checksum/download URL or the original file for ingested files, the inclusion of metadata about the parent collection(s) of an archived dataset version and use of the URL form of PIDs.
+- Display of archival status within the dataset page versions table, richer status options including success, pending, and failure states, with a complete API for managing archival status.
+- Support for batch archiving via API as an alternative to the current options of configuring archiving upon publication or archiving each dataset version manually.
+- Initial support for sending and receiving Linked Data Notification messages indicating relationships between a dataset and external resources (e.g. papers or other dataset) that can be used to trigger additional actions, such as the creation of a back-link to provide, for example, bi-dreictional linking between a published paper and a Dataverse dataset.
+
+
+
+## Major Use Cases and Infrastructure Enhancements
+
+Changes and fixes in this release include:
+
+- Administrators can configure an S3 store used in Dataverse to support users uploading/downloading files via Globus File Transfer (PR #8891)
+- Administrators can configure a RemoteOverlay store to allow files that remain hosted by a remote system to be added to a dataset. (PR #7325)
+- Administrators can configure dataverse to send archival Bag copies of published dataset versions to any S3-compatible service. (PR #8751)
+- Users can see information about a dataset's parent collection(s) in the OAI-ORE metadata export. (PR #8770)
+- Users and Administrators can use the OAI-ORE metadata export to retrieve and validate the checksum of the original file for ingested tabuar files. (PR #8901)
```

Comment:
The way this is worded... I'm not sure it's clear what the improvement is.

Simply stated (once everything is merged), bags contain the original file now, right? From tabular files, that is. Previously it was just the preservation tab separated version, from what I understand.

Update: I do see that this is covered below under "OAI-ORE and Archival Bag Changes". Maybe we should mention this here?

Reply:
Prior to this bags contained the ingested tab file  and the hash of the original file, so the tab file could not be validated/checked against the hash. Changed wording some - feel free to edit further.